code/modules/surgery/bodyparts/species_parts/genetics_bodyparts.dm

```diff
@@ -0,0 +1,222 @@
+#define BUNNY_HOP "bunny_hop"
+
+/datum/action/cooldown/spell/bunny_hop
```

Comment:
Wait what is this getting from being a spell that it doesn't just get from `action/cooldown`?